ext/mbstring/libmbfl/filters/mbfilter_iso8859_1.c

```diff
@@ -34,7 +34,7 @@
 #include "mbfilter.h"
 #include "mbfilter_iso8859_1.h"
 
-static const char *mbfl_encoding_8859_1_aliases[] = {"ISO_8859-1", "latin1", NULL};
+static const char *mbfl_encoding_8859_1_aliases[] = {"ISO_8859-1", "ISO8859-1", "ISO8859_1", "latin1", NULL};
```

Comment:
Apparently mbregex also supports variants like `ISO_8859_1`. Should those be added as well?

The number of permutations here is a bit annoying...

Reply:
> The number of permutations here is a bit annoying...

ACK. Since all aliases are under our control, we may consider to remove some of the schemes. At least the mixed ones (ISO_8859-*) look pretty strange to me.

Reply:
Another possibility might be to treat `-` and `_` the same when looking up encodings.

Reply:
Wouldn't `mb_encoding_aliases()` have to list all permutations then (let alone the docs)?